docs/src/pages/landing-page/QuasarComponents.vue

```diff
@@ -34,24 +42,53 @@
 import { defineComponent, ref, computed } from 'vue'
 import { components } from 'src/assets/landing-page/image-links.js'
 
-const FILTER_CHIPS = [ 'Buttons', 'Inputs', 'Loading', 'Media', 'Navigation', 'Panels & Popups', 'Scroll', 'Tables', 'Others' ]
+const OTHER_CATEGORY = 'other'
```

Comment:
I guess you splitted this so you could use it with the default category check
I think it's not needed anymore once you add categories explicitly on components